lib/src/summary.dart

```diff
@@ -120,8 +122,16 @@ class Summary extends Object with _$SummarySerializerMixin {
       this.suggestions,
       {this.flutterVersion});
 
-  factory Summary.fromJson(Map<String, dynamic> json) =>
-      _$SummaryFromJson(json);
+  factory Summary.fromJson(Map<String, dynamic> json) {
+    var panaVersion = new Version.parse(json['panaVersion']);
+    if (panaVersion.major == 0 && panaVersion.minor == 6) {
+      // Update the json in-place to have the expected values
+      var info = new DartSdkInfo.parse(json['sdkVersion']);
+      json['sdkVersion'] = info.version.toString();
```

Comment:
info.toJson() ?

Reply:
ah, it is not a @JsonSerializable... it could be though... if we are breaking things, we could do a full break as well :)

Reply:
`Version` is not in our package. We add support in w/ a custom TypeHelper in the tool dir.

I'd rather not churn thing...

Reply:
What I meant was that DartSdkInfo could be an object that we store in `Summary`.

Reply:
Ah...er...um...sure?

Reply:
We just want the SDK version in summary. The date/platform is just noise at the moment.

While I was parsing, I wanted to get everything.

Let's hold off until we want other things...at least that's my thought